cmake/Modules/FindPango.cmake

```diff
@@ -6,6 +6,38 @@
 
 include(LibFindMacros)
 
-libfind_pkg_check_modules(Pango_PKGCONF IMPORTED_TARGET pango)
-set(Pango_LIBRARY PkgConfig::Pango_PKGCONF)
-libfind_process(Pango)
+if(NOT WIN32)
```

Comment:
@orivej Does this still work on your system?
Due to #509 we kinda broke the windows build we had.. 
Windows mostlikely doesn't have pkg-config installed.. i'm looking for a way to combine both worlds